pom.xml

```diff
@@ -90,9 +90,4 @@
     </site>
   </distributionManagement>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
But is it good practice to override them? My other references as I'm learning this stuff is the AWS lambda plugin, which doesn't seem to set those properties: https://github.com/jenkinsci/aws-lambda-plugin/blob/master/pom.xml